specification/web/resource-manager/Microsoft.CertificateRegistration/2015-08-01/AppServiceCertificateOrders.json

```diff
@@ -1341,7 +1465,11 @@
         },
         "location": {
           "description": "Resource Location.",
-          "type": "string"
+          "type": "string",
+          "x-ms-mutability": [
```

Comment:
I know, we recommend to use `x-ms-mutability`. AutoRest SDK code generation doesn't use the extension for now but it may use it in the future. It's not an SDK breaking change. In the same time, removing `required` field may change an order of constructor parameters.